msteams-platform/messaging-extensions/build-api-based-message-extension.md

```diff
@@ -177,9 +217,36 @@ An API-based message extension is created.
 
 :::image type="content" source="../assets/images/Copilot/api-based-me-tdp-plugin-copilot.png" alt-text="Screenshot shows the plugin for copilot app created in the app features page in Teams Developer Portal.":::
 
+**Add Authentication**
+
+To add authentication to your messag extesion, follow these steps:
+
+1. Go to **Tools** > **API Key Registration**.
+
+1. In the **Register an API key** page, update the following:
+
+   1. **Description**:
+   1. **Add Domain**:
```

Comment:
Need to say more about this. What should the developer put here? How will Teams use it? What values does this allow? It says "domain" but the field takes a full URL (prefix?)

Reply:
Updated more information for Add domain and Description fields.